chart/templates/kafka-scada-json-deployment.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: kafka-scada-json-producer
+  name: kafka-scada-json-producer
+spec:
+  selector:
+    matchLabels:
+      app: kafka-scada-json-producer
+  template:
+    metadata:
+      labels:
+        app: kafka-scada-json-producer
+    spec:
+      containers:
+      - image: "{{ .Values.image }}:{{ .Values.imageTag }}"
+        command: 
+        - "python"
+        - "app/scada_parquet_transformer.py"
+        - "{{ .Values.clusterName }}-kafka-bootstrap:9093"
+        - {{ .Values.topics.consumer.name }}
+        - {{ .Values.topics.producer.name }}
```

These are arguments and I think it would be better to place them as such 
```suggestion
        args:
        - "{{ .Values.clusterName }}-kafka-bootstrap:9093"
        - {{ .Values.topics.consumer.name }}
        - {{ .Values.topics.producer.name }}
```